osf/models/preprint_service.py

```diff
@@ -210,16 +212,23 @@ def save(self, *args, **kwargs):
 
     def _send_preprint_confirmation(self, auth):
         # Send creator confirmation email
-        if self.provider._id == 'osf':
-            email_template = getattr(mails, 'PREPRINT_CONFIRMATION_DEFAULT')
-        else:
-            email_template = getattr(mails, 'PREPRINT_CONFIRMATION_BRANDED')(self.provider)
-
+        recipient = self.node.creator
+        event_type = utils.find_subscription_type('global_reviews')
+        user_subscriptions = get_user_subscriptions(recipient, event_type)
         mails.send_mail(
-            auth.user.username,
-            email_template,
-            user=auth.user,
-            node=self.node,
-            preprint=self,
-            osf_contact_email=settings.OSF_CONTACT_EMAIL,
+            recipient.username,
+            mails.REVIEWS_SUBMISSION_CONFIRMATION,
+            mimetype='html',
+            user=recipient,
+            **{
```

Comment:
personal preference, I'd throw this in a var above instead doing the dict literal in the function call